doc/release-notes.md

```diff
@@ -4,3 +4,56 @@ release-notes at release time)
 Notable changes
 ===============
 
+Mnemonic Recovery Phrases
+-------------------------
+
+The zcashd wallet has been modified to support ZIP 339 (derived from BIP 39)
+which describes how to derive the wallet's HD seed from a mnemonic phrase.
+The mnemonic phrase will be generated on load of the wallet, or the first time
+the wallet is unlocked, and is available via the z_exportwallet RPC call. All
+new addresses produced by the wallet are now derived from this seed using the
+HD wallet functionality described in ZIP 32 and ZIP 316. For users upgrading an
+existing Zcashd wallet, it is recommended that the wallet be backed up prior to
+upgrading to the 4.5.2 Zcashd release.
+
+Following the upgrade to 4.5.2, Zcashd will require that the user find their
+new emergency recovery phrase in the output of  the `z_exportwallet` RPC call, 
+and then verify that they have backed up the new recovery phrase by making a
+one-time call to the newly added `walletconfirmbackup` RPC method before the 
+wallet will allow the generation of new addresses. It is recommended that 
```

Comment:
Here we should remove direct references to the RPC methods, and instead say something to the effect of:

... will require that users confirm backup of their recovery phrase. This can be performed with tool X (insert details of how to use said CLI tool). GUI wallet developers can implement a similar confirmation workflow via the JSON-RPC interface following these guidelines (link to documentation inside the CLI tool that also serves this purpose).